### DISCUSSION TITLE
Code Table Request - add "ID/identifier" as an Agent type

### DISCUSSION THREAD

**Discussion by campmlc** [ArctosDB/arctos#6808] [Agent Committee]
## Initial Request

_**Goal**: We are expanding the usage of the Agents table to include identifiers, to provide IDs with capability for metadata, usage and activity tracking available to Agents. We need a new agent type of "identifer" or "ID" to accommodate this expanded usage.

_**Proposed Value**:  "identifier" or "ID/identifier".  I am asking for feedback on which is appropriate. The only issue with "identifier" as a value is that it could be confused with a person who makes a taxonomic identification.

_**Proposed Definition**:  Suggestions? "Any record identifier or entity which issues identifiers. Not to be confused with a person or agency providing taxonomic identifications. For example, "United States National Park Service Catalog".

_**Context**: See above.

_**Table**: https://arctos.database.museum/info/ctDocumentation.cfm?table=ctagent_type

_**Priority**: High, given the urgency with which the identifiers conversion is happening.

_**Project**: Add the issue to the [Code Table Management Project](https://github.com/ArctosDB/arctos/projects/13#card-31628184)._

_**Discussion**: Please reach out to anyone who might be affected by this change. Leave a comment or add this to the Committee agenda if you believe more focused conversation is necessary._

@ArctosDB/arctos-code-table-administrators 

## Approval

_All of the following must be checked before this may proceed._

_The [How-To Document](https://handbook.arctosdb.org/how_to/How-To-Manage-Code-Table-Requests.html) should be followed. Pay particular attention to terminology (with emphasis on consistency) and documentation (with emphasis on functionality). **No person should act in multiple roles**; the submitter cannot also serve as a Code Table Administrator, for example._

- [ ] Code Table Administrator[1] - check and initial, comment, or thumbs-up to indicate that the request complies with the how-to documentation and has your approval
- [ ] Code Table Administrator[2] - check and initial, comment, or thumbs-up to indicate that the request complies with the how-to documentation and has your approval
- [ ] DBA - The request is functionally acceptable. The term is not a functional duplicate, and is compatible with existing data and code.
- [ ] DBA - Appropriate code or handlers are in place as necessary. (ID_References, Media Relationships, Encumbrances, etc. require particular attention)

## Rejection

_If you believe this request should not proceed, explain why here. Suggest any changes that would make the change acceptable, alternate (usually existing) paths to the same goals, etc._

1. _Can a suitable solution be found here? If not, proceed to (2)_
2. _Can a suitable solution be found by Code Table Committee discussion? If not, proceed to (3)_
3. _Take the discussion to a monthly Arctos Working Group meeting for final resolution._

## Implementation

_Once all of the Approval Checklist is appropriately checked and there are no Rejection comments, or in special circumstances by decree of the Arctos Working Group, the change may be made._

_Review everything one last time.  Ensure the How-To has been followed. Ensure all checks have been made by appropriate personnel._

_Make changes as described above. Ensure the URL of this Issue is included in the definition._

_Close this Issue._

_**DO NOT** modify Arctos Authorities in any way before all points in this Issue have been fully addressed; data loss may result._

## Special Exemptions

_In very specific cases and by prior approval of The Committee, the approval process may be skipped, and implementation requirements may be slightly altered. Please note here if you are proceeding under one of these use cases._

1. _Adding an existing term to additional collection types may proceed immediately and without discussion, but doing so may also subject users to future cleanup efforts. If time allows, please review the term and definition as part of this step._
2. _The Committee may grant special access on particular tables to particular users. This should be exercised with great caution only after several smooth test cases, and generally limited to "taxonomy-like" data such as International Commission on Stratigraphy terminology._

**Comment by mkoo** [comment]
I dont see nor understand the need for another Agent type. "Identifier" is a role or action and we already have "person" "organization" (as well as bot and other which is just legacy)

Also, this has been a function of the agents table since the beginning, so rather than an expansion this is really using the the current model to its logically and most advantageous way to link appropriate agent to all their roles in Arctos.

In your proposal, does this mean I need to have 'Joseph Grinnell' as an agent and identifier?! 

Perhaps I dont understand what you mean. I am moving this to discussions but probably just need a conversation in realtime to understand
